data/meson.build

```diff
@@ -0,0 +1,52 @@
+libexecdir_config = configuration_data({
+    'libexecdir': libexec_dir,
+})
+
+# Systemd unit
+configure_file(
+    input: 'eos-metrics-event-recorder.service.in',
+    output: 'eos-metrics-event-recorder.service',
+    configuration: libexecdir_config,
+    install: true,
+    install_dir: systemd_dep.get_pkgconfig_variable('systemdsystemunitdir'),
+)
+
+# D-Bus service
+configure_file(
+    input: 'com.endlessm.Metrics.service.in',
+    output: 'com.endlessm.Metrics.service',
+    configuration: libexecdir_config,
+    install: true,
+    install_dir: get_option('datadir') / 'dbus-1' / 'system-services',
+)
+
+# D-Bus policy
+install_data(
+    'com.endlessm.Metrics.conf',
+    install_dir: get_option('datadir') / 'dbus-1' / 'system.d',
+)
+
+# tmpfiles rules
+configure_file(
+    input: 'eos-metrics.conf.in',
+    output: 'eos-metrics.conf',
+    configuration: configuration_data({
+        'configdir': config_dir,
+        'persistentcachedir': persistent_cache_dir,
+    }),
+    install: true,
+    install_dir: systemd_dep.get_pkgconfig_variable('tmpfilesdir'),
+)
+
+# Daemon configuration
+install_data(
+    'eos-metrics-permissions.conf',
+    install_dir: config_dir,
+    install_mode: ['rwxrwsr-x', 'metrics', 'metrics'],
```

Comment:
Given the config file is world-readable, does it have to be chowned to `metrics`? That seems a bit unusual. Every single file in `/etc` on my system is owned by `root`.

This is how the autotools system did things, though, so this is more of a passing comment/query than a request to change things.

Reply:
> the daemon modifies its own configuration in some situations

:scream:

That somehow feels wrong. But also definitely not something to address in this MR (and it might actually be the best approach anyway; I haven’t thought about it at all, and am just being reactionary).

Reply:
I had the same reaction!